app/services/support/data_exports/users_export.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Support
+  module DataExports
+    class UsersExport < Base
+      def id
+        "users"
+      end
+
+      def name
+        "All users"
+      end
+
+      def description
+        "The list of all users from current recruitment cycle with columns: provider_code, provider_name, provider_type, first_name, last_name, email_address"
```

Comment:
```suggestion
        "The list of all users affiliated with a provider from current recruitment cycle with columns: provider_code, provider_name, provider_type, first_name, last_name, email_address"
```

Why are we isn't this in translation?